From 2150fa0cbba71f12ce27f9ce123fa8c7cd939e07 Mon Sep 17 00:00:00 2001
From: Zumeng Chen <zumeng.chen@windriver.com>
Date: Mon, 13 Feb 2017 12:13:14 +0800
Subject: [PATCH 3/6] arm64/dts/xilinx: remove unstable phy interfaces

Currently xilinx phys ralated designs are still not stable, so remove them and
usb & sata hardisk works well.

Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 arch/arm64/boot/dts/xilinx/zynqmp-zcu102.dts |    2 --
 1 files changed, 0 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/xilinx/zynqmp-zcu102.dts b/arch/arm64/boot/dts/xilinx/zynqmp-zcu102.dts
index e1c6816..b3484b8 100644
--- a/arch/arm64/boot/dts/xilinx/zynqmp-zcu102.dts
+++ b/arch/arm64/boot/dts/xilinx/zynqmp-zcu102.dts
@@ -890,7 +890,6 @@ drivers/hwmon/pmbus/Makefile:11:obj-$(CONFIG_SENSORS_MAX20751)  += max20751.o
 	ceva,p1-burst-params = /bits/ 8 <0x13 0x08 0x4A 0x06>;
 	ceva,p1-retry-params = /bits/ 16 <0x96A4 0x3FFC>;
 	phy-names = "sata-phy";
-	phys = <&lane3 PHY_TYPE_SATA 1 1 125000000>;
 };
 
 /* SD1 with level shifter */
@@ -930,7 +929,6 @@ drivers/hwmon/pmbus/Makefile:11:obj-$(CONFIG_SENSORS_MAX20751)  += max20751.o
 	dr_mode = "host";
 	snps,usb3_lpm_capable;
 	phy-names = "usb3-phy";
-	phys = <&lane2 PHY_TYPE_USB3 0 2 26000000>;
 };
 
 &xilinx_ams {
-- 
1.7.5.4

